docs/src/main/sphinx/connector/row-decoding.fragment

```diff
@@ -0,0 +1,323 @@
+Row decoding
+^^^^^^^^^^^^
+
+For key and message, a decoder is used to map message and key data onto table
+columns.
+
+The connector contains the following decoders:
```

Comment:
Depending on answer from @nevillelyh to my question it might make sense to remove this list since the list can vary per connector.

In such case instead of having a single row-decoding.fragment it might make more sense to have one per decoder like raw-decoder.fragment and so on and then use which ones are applicable to some connector.

Reply:
Let's keep the list as it is.